strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/artifact/locator/handlers/AbstractArtifactLocationHandler.java

```diff
@@ -19,42 +24,47 @@
 public abstract class AbstractArtifactLocationHandler
         implements ArtifactDirectoryOperation
 {
-
+    private static final Logger logger = LoggerFactory.getLogger(AbstractArtifactLocationHandler.class);
+    
     private Storage storage;
-
-    private Repository repository;
-
-    private FilenameFilter filter;
-
-    private LinkedHashMap<String, List<File>> visitedRootPaths = new LinkedHashMap<>();
+    private LinkedHashMap<RepositoryPath, List<RepositoryPath>> visitedRootPaths = new LinkedHashMap<>();
```

Comment:
Cosmetic: Could you please, leave empty lines:
* Between field declarations
* Before and after logging blocks and `if`-statements and loops
* Before `return` statements
* At the end of files

Thanks! :)